### ISSUE TITLE
Eventlocation "online"

### ISSUE THREAD

**Issue by Naadjjaa** [stadtluzern/decidim-ocl#62]
Wenn ein Event online stattfindet, muss dieser nicht auf der Karte auftauchen. Wie könnten wir dies umsetzen?

**Comment by Kagemaru** [comment]
Könnte man bei der Adresse einfach "Online" oder "Zoom" eintragen?

**Comment by Naadjjaa** [comment]
Ja, aber: (1) die Adresse ist ein Pflichtfeld und (2) es geht auch darum, dass die Onlineveranstaltungen die Punkte auf der Übersichts-Karte verfälschen.

**Comment by carlobeltrame** [comment]
Inwiefern verfälscht es denn die Punkte auf der Übersichts-Karte? Wo tauchen diese "Online" Punkte auf? Wenn wir uns auf einen oder zwei spezielle Werte ("Online", "Zoom", etc.) einigen die nicht auf der Karte angezeigt werden sollen, könnten wir das irgendwo in der Nähe des Adress-Felds für den Nutzer anschreiben und solche Events auf der Karte einfach rausfiltern.

**Comment by innosmith** [comment]
@Naadjjaa, @Kagemaru, @carlobeltrame : eben einen spannenden Tipp von einem "Decidim Awesome" Entwickler erhalten, diese Anforderung rein mit einer CSS-Anpassung zu lösen. Wenn ein Event online ist, verbergen die Kollegen die Adresse. Ich sende euch dazu die Email weiter, da ich den Text hier nicht einfügen kann.

**Comment by innosmith** [comment]
Mit v0.24 gelöst https://github.com/decidim/decidim/pull/6572
